app/src/main/java/org/thoughtcrime/securesms/home/PathActivity.kt

```diff
@@ -111,6 +111,7 @@ class PathActivity : PassphraseRequiredActionBarActivity() {
             for (row in rows) {
                 binding.pathRowsContainer.addView(row)
             }
+            binding.pathAttribution.visibility = View.VISIBLE
```

Comment:
Should we hide this in the opposite case?

Reply:
I don't know if there is a case where you can lose all paths while on the screen after you've successfully gotten paths but it probably doesn't hurt to add it